diff --git a/lib/client.js b/lib/client.js
index 2b3050ee89af0c6f8fcfc77a299a4fa615cecf28..3956645ca448d03032b1de32a60fd20b2b5e6b48 100644
--- a/lib/client.js
+++ b/lib/client.js
@@ -288,6 +288,7 @@ window.__ModuleLoader__.load({
 
       ctx.slots.inject('settings.plugin.item', () => ctx.slots.register({
         name: 'settings.plugin.item',
+        key: 'dsh-web-search',
         id: 'dsh-web-search',
         order: 22,
         locale: 'dsh-web-search',
